docs/examples/basic-price-oracle/main.go

```diff
@@ -174,12 +226,78 @@ func startService(grpcServer *grpc.Server) error {
 	return grpcServer.Serve(grpcListener)
 }
 
+// Generate a self-signed TLS certificate and private key.
+func generateSelfSignedCert() (tls.Certificate, error) {
+	privateKey, err := ecdsa.GenerateKey(elliptic.P256(), rand.Reader)
```

Comment:
Would other curves be more desired? p256 vs https://pkg.go.dev/crypto/elliptic#P521

Reply:
This is just to get TLS working with the example service. Its just example code and not security critical.